src/components/calcite-value-list/calcite-value-list.tsx

```diff
@@ -72,9 +72,9 @@ export class CalciteValueList<
   @Prop({ reflect: true }) filterPlaceholder: string;
 
   /**
-   * If this is set and drag is enabled, items can be dropped between lists of the same group.
+   * To drag elements from one list into another, both lists must have the same group value.
```

Comment:
I like your phrasing better. Can we also add some text to describe the prop? Something like:

```
/**
 * The list's group name. 
 * 
 *  To drag items between lists, both must be draggable and have the same group value.
 */
```

?

Reply:
I think its more of an id than a name, right? does name imply that it will be rendered?

Reply:
Either would work. I remember seeing "name" used more in the group prop context.